frontend/src/modules/controllers/ValueSlider.tsx

```diff
@@ -69,6 +73,20 @@ const StyledSlider = withStyles({
   },
 })(Slider);
 
+/**
+ * @typedef Props
+ *
+ * Props interface for the slider adjustments
+ *
+ * @prop {number} min Minimum Range of Alarm
+ * @prop {number} max Minimum Range of the Slider
+ * @prop {function} onChange Callback on slider value change
+ * @prop {number[]} rangeValues Slider [minValue,maxValue] range value
```

Comment:
What's the relationship between `min`, `max`, and `rangeValues`? Do `min` and `max` specify the positions of the circles in the sliders, while `rangeValues` specifies the smallest and largest allowed values for `min` and `max`, or vice versa?

Reply:
Yes Vice versa, `rangeValues` are used to specify the positions of the circles in the sliders. `min` & `max` are used to specifies the smallest and largest allowed values